### ISSUE TITLE
Missing TrustedDeviceManagerInterface

### ISSUE THREAD

**Issue by simondaigre** [scheb/2fa#121]
**Bundle version**: 6.0.0
**Symfony version**: 5.4.2
**PHP version**: 8.0.14
**Using authenticators** (`enable_authenticator_manager: true`): YES

**Description**
Hi,
Got this error when I try your bundle with Opcache preload enabled.
I didnt find the mentioned interface in your bundle and I dont use TrustedDevice feature/bundle.

`PHP Fatal error:  Failed to load class Scheb\TwoFactorBundle\Security\TwoFactor\Trusted\TrustedDeviceManagerInterface used by typed property Scheb\TwoFactorBundle\Controller\FormController::$trustedDeviceManager during preloading`

**Comment by scheb** [comment]
I've found issues symfony/symfony#44380 and symfony/symfony#44364 which read to me as if it's a known problem with the preloader, when typed properties are used (which the bundle does since version 6).

Though I'm unwilling to remove the type hints from the properties and make my code worse, for the sake of working around an issue that isn't caused by the bundle.

Some solutions based on what I've learned so far:
- Use bundle version 5, which doesn't have typed properties and therefore doesn't have the problem
- Use PHP 8.1, which handles typed properties differently in preloading and therefore doesn't have the problem, [as stated in this comment](https://github.com/symfony/symfony/issues/44364#issuecomment-982840815). But as also stated, ["preloading in general, the usefulness is greatly diminished with 8.1"](https://github.com/symfony/symfony/pull/44380#issuecomment-996174208), so you might want to skip it anyways when using PHP 8.1.
- Install `scheb/2fa-trusted-device` package even when you're not using it, to solve the undefined class requirement for the preloader

**Comment by simondaigre** [comment]
Thanks for you quick answer. I will remove preloading so (or maybe give a try of the remember device bundle)

**Comment by scheb** [comment]
Ah good point, the `TrustedDeviceManagerInterface` is part of the optional package. You don't have it installed, so the preloader can't find the class. Another solution would be to just install `scheb/2fa-trusted-device` package even when you're not using it, to solve the undefined class requirement for the preloader (adding this above as another solution).

I tried to reproduce the error locally, but couldn't get it to fail. When does that "PHP Fatal error" happen for you? I tried `cache:clear --env=prod`, but that didn't error for me.
